Makefile

```diff
@@ -49,24 +49,4 @@ clean:
 
 .PHONY: install
 install:
-	go get -u github.com/gogo/protobuf/protoc-gen-gogofaster
-	go get -u github.com/gogo/protobuf/gogoproto
-	go get -u github.com/golang/protobuf/protoc-gen-go
-	go get -u github.com/tinylib/msgp
-	go get -u github.com/andyleap/gencode
-	go get -u github.com/mailru/easyjson/...
-	go get -u github.com/DeDiS/protobuf
-	go get -u github.com/Sereal/Sereal/Go/sereal
-	go get -u github.com/alecthomas/binary
-	go get -u github.com/davecgh/go-xdr/xdr
-	go get -u github.com/gogo/protobuf/proto
-	go get -u github.com/google/flatbuffers/go
-	go get -u github.com/tinylib/msgp/msgp
-	go get -u github.com/ugorji/go/codec
-	go get -u gopkg.in/mgo.v2/bson
-	go get -u gopkg.in/vmihailenco/msgpack.v2
-	go get -u github.com/golang/protobuf/proto
-	go get -u github.com/hprose/hprose-go/io
-	go get -u github.com/pascaldekloe/colfer/cmd/colf
-	go get -u github.com/calmh/xdr
-	go get -u github.com/niubaoshu/gotiny
+	go get -u
```

Comment:
Go Modules should automatically install everything. `go get -u` was left to update all dependencies to the latest version.